Factorial.rb

```diff
@@ -0,0 +1,23 @@
+=begin 
+Given n of 1 or more, return the factorial of n, which is n * (n-1) * (n-2) ... 1. Compute the result recursively (without loops).
+factorial(1) → 1
+factorial(2) → 2
+factorial(3) → 6
+=end
+
+def factorial(number)
+    store = 1
+    return 1 if number == 1
+     
+   
+    store *= number
+    return store * factorial(number - 1)

```

Comment:
```suggestion
   store * number * factorial(number - 1)
```

Would this not be the same?